rest-api/tools/import_organizations.py

```diff
@@ -194,10 +212,17 @@ def _entity_from_row(self, row):
 
   def _update_entity(self, entity, existing_entity, session, dry_run):
     self._populate_lat_lng_and_time_zone(entity, existing_entity)
+    if entity.siteStatus == self.ACTIVE and (entity.latitude == None or entity.longitude == None):
+      self.errors.append('Skipped active site without geocoding: {}'.format(entity.googleGroup))
+      self.skip = True
```

Comment:
Rather than having a "skip" field that you have to set here and clear elsewhere, can we just have this method returning None indicate we skipped the entity?

Reply:
changed to return a tuple since we used None for match count.